cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -647,12 +662,15 @@ def create_quantum_job(
                     to a retryable error and retry attempts failed.
             ValueError: If the parameters are invalid.
         """
-        # Wrap the transport method to add retry and timeout logic.
+        # Wrap the transport method to add retry, timeout, and compression logic.
         if 'create_quantum_job' not in self._inner_api_calls:
             self._inner_api_calls[
                 'create_quantum_job'
             ] = google.api_core.gapic_v1.method.wrap_method(
-                self.transport.create_quantum_job,
+                functools.partial(
```

Comment:
Should this be done in a helper to assist with this wrapping readability and future maintenance? It's repeated many times.

Reply:
Will do - with the other changes this becomes especially necessary.